docs/getting-started.md

```diff
@@ -82,7 +153,7 @@ Instead, let's do it as a step in the Production Release process (or manually).
   dbt run-operation create_masking_policies --args '{debug: true}'
```

Comment:
Sorry not this doc, but the actual macro `create_masking_policies` (also `create_tags` one), we have the duplications in the executed script:
![image](https://github.com/user-attachments/assets/23019c0c-a15b-458a-b72d-cce4fef92852)

Not sure if we can do something here to remove dups, or I'm happy to fix in the next PR too

Reply:
Same issue for `apply_mps_to_tags`

Reply:
This one was an "issue" before I started working on my features, so I didn't bother trying to fix it in this PR. However, I have just resolved it now.
I removed some of the information from the tag comment because it was related to the specific column it was found on in the config, however if it is applied to multiple columns, this is fairly useless info. 
Also, not sure how useful the target name is in the comment of the tag either?